.github/workflows/release.yml

```diff
@@ -260,6 +260,17 @@ jobs:
           console.log(releaseNote)
           core.setOutput('version', runnerVersion);
           core.setOutput('note', releaseNote);
+
+    - name: Validate Packages HASH
+      working-directory: _package
```

Comment:
Not sure how the artifacts got corrupted in the first place, so this might be irrelevant, but should we validate the trim files in `_package/trim*/**` folders?

Reply:
the runner will validate the hash when it's try to use any of the trimmed packages, it will fall back to the full package if the validation failed.
So we only need to validate hash for the full packages as a minimum requirement.